samples/glean/src/main/java/org/mozilla/samples/glean/MainActivity.kt

```diff
@@ -59,7 +61,7 @@ open class MainActivity : AppCompatActivity(), ExperimentUpdateReceiver.Experime
         SamplesGleanLibrary.recordMetric()
         SamplesGleanLibrary.recordExperiment()
 
-        // The following is not relevant to the Glean SDK, but to the experiments library.
+        // The following is not relevant to the Glean SDK, but to the Nimbus experiments library.
```

Comment:
Can you mark the end of the block of code that's only relevant to Nimbus? (here and in other places in the sample)

Reply:
I refactored a bit to try and put all of the Nimbus relevant code into their own functions, as well as including "begin" and "end" comments to indicate that the code was only relevant to Nimbus.